lib/kafka.rb

```diff
@@ -225,6 +225,15 @@ class OffsetCommitError < Error
   class FetchError < Error
   end
 
+  class NoPartitionsAssignedError < Error
+  end
```

Comment:
How did this sneak in here?

Reply:
I have no idea... Maybe some merge i missed.